sdk/securityinsights/Azure.ResourceManager.SecurityInsights/src/Generated/Models/Entity.cs

```diff
@@ -7,34 +7,33 @@
 
 using Azure.Core;
 using Azure.ResourceManager.Models;
-using Azure.ResourceManager.SecurityInsights.Models;
 
-namespace Azure.ResourceManager.SecurityInsights
+namespace Azure.ResourceManager.SecurityInsights.Models
 {
     /// <summary>
-    /// A class representing the Entity data model.
-    /// Please note <see cref="EntityData"/> is the base class. According to the scenario, a derived class of the base class might need to be assigned here, or this property needs to be casted to one of the possible derived classes.
-    /// The available derived classes include <see cref="AccountEntity"/>, <see cref="AzureResourceEntity"/>, <see cref="HuntingBookmark"/>, <see cref="CloudApplicationEntity"/>, <see cref="DnsEntity"/>, <see cref="FileEntity"/>, <see cref="FileHashEntity"/>, <see cref="HostEntity"/>, <see cref="IoTDeviceEntity"/>, <see cref="IPEntity"/>, <see cref="MailClusterEntity"/>, <see cref="MailMessageEntity"/>, <see cref="MailboxEntity"/>, <see cref="MalwareEntity"/>, <see cref="NicEntity"/>, <see cref="ProcessEntity"/>, <see cref="RegistryKeyEntity"/>, <see cref="RegistryValueEntity"/>, <see cref="SecurityAlert"/>, <see cref="SecurityGroupEntity"/>, <see cref="SubmissionMailEntity"/> and <see cref="UrlEntity"/>.
+    /// Specific entity.
+    /// Please note <see cref="Entity"/> is the base class. According to the scenario, a derived class of the base class might need to be assigned here, or this property needs to be casted to one of the possible derived classes.
+    /// The available derived classes include <see cref="AccountEntity"/>, <see cref="AzureResourceEntity"/>, <see cref="HuntingBookmark"/>, <see cref="CloudApplicationEntity"/>, <see cref="DnsEntity"/>, <see cref="FileEntity"/>, <see cref="FileHashEntity"/>, <see cref="HostEntity"/>, <see cref="IoTDeviceEntity"/>, <see cref="IPEntity"/>, <see cref="MailClusterEntity"/>, <see cref="MailMessageEntity"/>, <see cref="MailboxEntity"/>, <see cref="MalwareEntity"/>, <see cref="ProcessEntity"/>, <see cref="RegistryKeyEntity"/>, <see cref="RegistryValueEntity"/>, <see cref="SecurityAlert"/>, <see cref="SecurityGroupEntity"/>, <see cref="SubmissionMailEntity"/> and <see cref="UrlEntity"/>.
     /// </summary>
-    public partial class EntityData : ResourceData
+    public partial class Entity : ResourceData
```

Comment:
This would be a breaking change that is hard to deal with when we change back to preview version as it has lots of derived classes. Please check if we can use customized code to rename it to `EntityData` and move it to the `Azure.ResourceManager.SecurityInsights` namespace in stable version.